desktop/core/src/desktop/templates/hue.mako

```diff
@@ -55,8 +55,25 @@
         send_page_view: false, 
         page_location: 'redacted',
         page_referrer: 'redacted',
-        allow_google_signals: false
+        allow_google_signals: false,
+        // Make GA cookies session-based (expire when browser closes)
+        cookie_flags: 'secure;samesite=strict',
+        anonymize_ip: true,
+        storage: 'none'
```

Comment:
I think you can set the `cookie_expires` parameters to 0 for it to expire once the current browser session ends.